### ISSUE TITLE
22447486: watchOS 2: Calendar doesn't have enough contrast

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#5882]
#### Description

Summary:
The calendar app needs way more contrast - the detail text on these events fails accessibility requirements for contrast.

Steps to Reproduce:
1. Open the Calendar app to view a calendar appointment on a green-tinted event.

Expected Results:
I'd expect that Apple would have contrast settings that pass accessibility requirements.

Actual Results:
The subtitle text on events isn't legible (see screenshot)

Version:
13S5325c

Notes:

Configuration:

Attachments:
'Napkin 08-26-15, 4.29.18 PM.png' was successfully uploaded.

For OpenRadar folks: Here's the attachment: http://clrk.it/19x1G
## 

Product Version: build 13S5325c
Created: 2015-08-26T23:34:57.428130
Originated: 2015-08-26T00:00:00
Open Radar Link: http://www.openradar.me/22447486
